LEAF_Nexus/sources/Position.php

```diff
@@ -845,6 +876,10 @@ public function removePermission($positionID, $categoryID, $UID, $permissionType
      */
     public function deletePosition($positionID)
     {
+        if(!is_numeric($positionID)) {
+            throw new Exception('invalid input');
```

Comment:
Since the method only wants to know if something was deleted, returning `0` is sufficient. If it doesn't exist, a delete didn't work. Throwing the `Exception` can be omitted.